File: led.c
#include <stdio.h>

int main(){
    

    int num, aux = 0;
    int cont = 0;
    scanf("%d", &num);

    

    while(aux < num){
        char led[100];

        fgets(led, 100, stdin);

        
        for(int i = 0; i < contadorString(led); i++){
            if (led[i] == '1'){
                cont = cont + 2;
            } else if (led[i] == '2'){
                cont = cont + 5;
            } else if (led[i] == '3'){
                cont = cont + 5;
            } else if (led[i] == '4') {
                cont = cont + 4;
            } else if (led[i] == '5') {
                cont = cont + 5;
            } else if (led[i] == '6') {
                cont = cont + 6;
            } else if (led[i] == '7') {
                cont = cont + 3;
            } else if (led[i] == '8') {
                cont = cont + 7;
            } else if (led[i] == '9') {
                cont = cont + 6;
            } else if (led[i] == '0') {
                cont = cont + 6;
            }
        }
        printf("%d leds", cont);
        aux++;
    }
}

int contadorString(char *s){
    int i = 0;
    int cont = 0;
    while(s[i] != '\0'){
        cont++;
        i++;
        
    }
    return cont - 1;
}

File: led.py


def contadorString(nome:str):
    cont = 0
    for i in nome:
        cont = cont + 1
        
    return cont


#print(contadorString("Tiago"))
#nome = "Tiago"
#print(nome[0])


aux = 0
cont = 0

num = int(input())

while aux < num:
    cont = 0
    led = input()
    
    for i in range(contadorString(led)):
        if led[i] == '1':
            cont = cont + 2
        elif led[i] == '2':
            cont = cont + 5
        elif led[i] == '3':
            cont = cont + 5
        elif led[i] == '4':
            cont = cont + 4
        elif led[i] == '5':
            cont = cont + 5
        elif led[i] == '6':
            cont = cont + 6
        elif led[i] == '7':
            cont = cont + 3
        elif led[i] == '8':
            cont = cont +7
        elif led[i] == '9':
            cont = cont + 6
        elif led[i] == '0':
            cont = cont + 6
            
    print(f'{cont} leds')
    aux = aux + 1
